libtelemetry/src/full/java/com/mapbox/android/telemetry/errors/TokenChangeBroadcastReceiver.java

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.android.telemetry.crash;
```

Comment:
Noting that this is technically breaking Semver as packages of a `public` class changed